client/src/pages/lessons/edit.tsx

```diff
@@ -894,194 +651,299 @@ const LessonEdit = (props: {
                       variant="outlined"
                     />
                   </Grid>
+                  <Grid item xs={12}>
+                    <Grid
+                      container
+                      // TODO: With MUI v5 We can dimply do: rowSpacing={2}
+                      direction={"row"}
+                      spacing={2}
+                    >
+                      <Grid item xs={6}>
+                        <TextField
+                          data-cy="video-start"
+                          label="Video Start Time"
+                          placeholder="0.0"
+                          type="number"
+                          required
+                          multiline
+                          maxRows={1}
+                          // style={{ width: "50%" }}
+                          InputLabelProps={{
+                            shrink: true,
+                          }}
+                          fullWidth
+                          value={
+                            lessonUnderEdit.lesson.media &&
+                            lessonUnderEdit.lesson.media.props
+                              ? parseFloat(
+                                  getProp(
+                                    lessonUnderEdit.lesson.media.props,
+                                    "start"
+                                  )
+                                ) || 0
+                              : 0
+                          }
+                          onChange={(e) => {
```

Comment:
should move this anonymous function into its own separate named function. rule of thumb is if the anonymous function is more than 2-3 lines of code, move it into it's own name function. (apply this to other anonymous functions in the file)